app/policies/portfolio_policy.rb

```diff
@@ -30,4 +30,16 @@ def copy?
 
     true
   end
+
+  def share?
+    rbac_access.admin_check
```

Comment:
@eclarizio Shouldn't the Policy actions just return a true/false and have the Pundit authorization error from here do the right thing
https://github.com/RedHatInsights/catalog-api/blob/97ef46ab233afac254b4fa5cc1cb9a558ddb8b8b/config/initializers/custom_exception_mappings.rb#L22
we are raising our own exception Catalog::NotAuthorized and having this function always return a true if an exception is not raised. It would be cleaner if we just returned a true/false here and let Pundit handle the rest.

Reply:
@mkanoor I agree, I think this was just the pattern I established. I can change these but I think it's probably best for a separate PR because it's going to touch a lot of actions that this PR doesn't care about.